db/schema.rb

```diff
@@ -132,14 +132,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
-  create_table "users_roles", id: false, force: :cascade do |t|
```

Comment:
I'm not sure why this was this table was removed from schema.rb, but I don't think we want that table removed from schema.rb.

Reply:
Hey @rossfuhrman! I'm not sure how this happen , because I didn't remove it manually